mysql_tests/test_loader.py

```diff
@@ -0,0 +1,402 @@
+import random
+from datetime import datetime
+
+import pytest
+from async_generator import yield_, async_generator
+
+from gino.loader import AliasLoader
+from sqlalchemy import select
+from sqlalchemy.sql.functions import count
+from .models import (
+    db,
+    User,
+    Team,
+    TeamWithDefaultCompany,
+    TeamWithoutMembersSetter,
+    Company,
+    CompanyWithoutTeamsSetter,
+)
+
+pytestmark = pytest.mark.asyncio
+
+
+@pytest.fixture
+@async_generator
+async def user(bind):
+    c = await Company.create()
+    t1 = await Team.create(company_id=c.id)
+    t2 = await Team.create(company_id=c.id, parent_id=t1.id)
+    t3 = await Team.create(company_id=c.id, parent_id=t1.id)
+    u = await User.create(team_id=t2.id)
+    u.team = t2
+    t2.parent = t1
+    t2.company = c
+    t1.company = c
+    await yield_(u)
+    await User.delete.gino.status()
+    await Team.delete.gino.status()
+    await Company.delete.gino.status()
+
+
+async def test_model_alternative(user):
+    u = await User.query.gino.load(User).first()
+    assert isinstance(u, User)
+    assert u.id == user.id
+    assert u.nickname == user.nickname
+
+
+async def test_scalar(user):
+    name = await User.query.gino.load(User.nickname).first()
+    assert user.nickname == name
+
+    uid, name = await User.query.gino.load((User.id, User.nickname)).first()
+    assert user.id == uid
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/python-gino/gino/pullRequest?prid=5587950)